dbt/adapters/spark_cde/cdeapisession.py

```diff
@@ -101,57 +106,112 @@ def execute(self, sql: str, *parameters: Any) -> None:
 
         # 0. generate a job name
         job_name = self.generate_job_name()
+        logger.debug(
+            "{}: Job created with id: {} for SQL statement:\n{}".format(
+                job_name, job_name, sql
+            )
+        )
 
         # 1. create resource
-        self._cde_connection.delete_resource(job_name)
+        logger.debug("{}: Create resources: files".format(job_name))
         self._cde_connection.create_resource(job_name, "files")
+        logger.debug("{}: Done create resource: files".format(job_name))
 
+        # 2. upload the resources
         sql_resource = self._cde_api_helper.generate_sql_resource(job_name, sql)
         py_resource = self._cde_api_helper.get_python_wrapper_resource(sql_resource)
 
-        # 2. upload the resource
+        logger.debug(
+            "{}: Upload resource: SQL resource: {}".format(
+                job_name, sql_resource["file_name"]
+            )
+        )
         self._cde_connection.upload_resource(job_name, sql_resource)
+        logger.debug(
+            "{}: Done upload resources: SQL resource: {}".format(
+                job_name, sql_resource["file_name"]
+            )
+        )
+        logger.debug(
+            "{}: Upload resource: py resource: {}".format(
+                job_name, py_resource["file_name"]
+            )
+        )
         self._cde_connection.upload_resource(job_name, py_resource)
 
-        # 2. submit the job
-        self._cde_connection.delete_job(job_name)
+        logger.debug(
+            "{}: Done upload resource: py resource: {}".format(
+                job_name, py_resource["file_name"]
+            )
+        )
 
+        # 3. submit the job
+        logger.debug("{}: Submit job".format(job_name))
         self._cde_connection.submit_job(job_name, job_name, sql_resource, py_resource)
+        logger.debug("{}: Done submit job".format(job_name))
 
+        # 4. run the job
+        logger.debug("{}: Run job".format(job_name))
         job = self._cde_connection.run_job(job_name).json()
-        self._cde_connection.get_job_status(job_name)
+        logger.debug("{}: Done run job".format(job_name))
 
-        # 3. run the job
+        # 5. wait for the result
+        logger.debug("{}: Get job status".format(job_name))
         job_status = self._cde_connection.get_job_run_status(job).json()
-        # 4. wait for the result
+        logger.debug(
+            "{}: Current Job status: {}".format(job_name, job_status["status"])
+        )
+        logger.debug("{}: Done get job status".format(job_name))
+
         while job_status["status"] != CDEApiConnection.JOB_STATUS["succeeded"]:
+            logger.debug("{}: Sleep for {} seconds".format(job_name, DEFAULT_POLL_WAIT))
             time.sleep(DEFAULT_POLL_WAIT)
+            logger.debug(
+                "{}: Done sleep for {} seconds".format(job_name, DEFAULT_POLL_WAIT)
+            )
+
+            logger.debug("{}: Get job status".format(job_name))
```

Comment:
any reason to have 3 logs here..is this also printing the timestamp, is the gaol to get timing data out of this. Same comment for other logs as well.

Reply:
Yes the utc times will be in the log. So we don't need to manually time this.

For e.g. for create resource method
./dbt.log:41:03:23:34.699497 [debug] [ThreadPool]: Spark adapter: dbt-job-1662089014699-00000532: Create resources: files
./dbt.log:42:03:23:35.553290 [debug] [ThreadPool]: Spark adapter: dbt-job-1662089014699-00000532: Done create resource: files